starter-project-backend-g31/src/controllers/comment.ts

```diff
@@ -38,6 +47,12 @@ export async function addComment(req: Request,res:Response){
 }
 
 export async function deleteCommentById(req:Request,res:Response){
+
+    const user = await userModel.findById(req.params.userId);

```

Comment:
it would be better to check if the actual user who created the comment should delete the comment, 
check user._id equlaity